packages/arb-token-bridge-ui/src/components/TransferPanel/Routes/Route.tsx

```diff
@@ -0,0 +1,206 @@
+  }
+
+  switch (badgeType) {
+    case 'security-guaranteed': {
+      return {
+        bgClassname: 'bg-lime-dark',
+        textClassname: 'text-lime',
+        text: 'Security guaranteed by Arbitrum'
+      }
+    }
+  }
+}
+
+export function Route({
+  type,
+  bridge,
+  bridgeIconURI,
+  durationMs,
+  amountReceived,
+  isLoadingGasEstimate,
+  overrideToken,
+  gasCost,
+  gasToken,
+  tag,
+  onRouteSelected,
+  selected
+}: RouteProps) {
+  const [networks] = useNetworks()
+  const { childChainProvider } = useNetworksRelationship(networks)
+  const childNativeCurrency = useNativeCurrency({
+    provider: childChainProvider
+  })
+  const [_token] = useSelectedToken()
+
+  const token = overrideToken || _token || childNativeCurrency
+
+  const { name, icon, width, height } = getBridgeConfigFromType(type)
+  const { bgClassname, textClassname, text } = getBadgeProps(tag)
+
+  return (
+    <div
+      className={twMerge(
+        'group cursor-pointer rounded border border-[#ffffff33] text-white transition-colors',
+        selected && 'border-white'
+      )}
+      onClick={() => onRouteSelected(type)}
+    >
+      <div className="bg-gray-8 flex items-center rounded-t px-4 py-2 text-xs">
+        <Image
+          src={icon}
+          width={width}
+          height={height}
+          alt="protocol"
+          className="mr-1"
+        />
+        Powered by {name}
+      </div>
+      <div
+        className={twMerge(
+          'relative flex rounded-b bg-[#303030] px-4 py-3 text-sm transition-colors group-hover:bg-[#474747]',
+          selected && 'bg-[#474747]'
+        )}
+      >
+        <div className="flex flex-col">
+          <span>You will receive:</span>
+          <div className="flex items-center text-lg">
+            {'logoURI' in token ? (
+              <SafeImage
+                src={token.logoURI}
+                width={15}
+                height={15}
+                alt="bridge"
+                className="mr-1 rounded-full"
+              />
+            ) : null}
+            {formatAmount(BigNumber.from(amountReceived), {
+              decimals: token.decimals,
+              symbol: token.symbol
+            })}
+          </div>
+        </div>
+        <div className="ml-6 flex flex-col justify-between gap-3">
+          <div className="flex items-center">
+            <Image
+              src="/icons/duration.svg"
+              width={15}
+              height={15}
+              alt="duration"
+            />
+            <span className="ml-1">
+              {dayjs().add(durationMs, 'millisecond').fromNow(true)}
+            </span>
+          </div>
+          <div className="flex items-center">
+            <Image src="/icons/gas.svg" width={15} height={15} alt="gas" />
+            <span className="ml-1">
+              {isLoadingGasEstimate ? (
+                <Loader size="small" color="white" />
+              ) : gasCost ? (
+                formatAmount(BigNumber.from(gasCost), {
```

Comment:
Are we okay with removing the feature of showing the fiat/dollar value of the gas fee? IMHO it's a pretty useful feature.